### PR TITLE
Say ‘UK government’ sent the alert

### PR DESCRIPTION
**Pull request by quis**
This is consistent with who we are saying sent the alert in comms

> **government**
> Lower case unless it’s a full title. For example: ‘UK government’, but ‘Her Majesty’s Government of the United Kingdom of Great Britain and Northern Ireland’.

— https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#government

### CODE REVIEW

File: src/alert.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h1 class="govuk-heading-xl alerts-icon__container alerts-icon__container--48">
       {{ alert_data.description | paragraphize }}
 
       <p class="govuk-body govuk-!-margin-bottom-0">
-        Issued by Cabinet Office <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
+        Issued by the UK government <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
```
Comment (idavidmcdonald) [inline src/alert.html:57]: Is it `Issued by the UK government` or is it `Issued by UK Government`? I assume the former but worth checking with @karlchillmaid

Comment (karlchillmaid) [reply src/alert.html:57]: It should be `the`... also, I prefer `sent` to `issued`.

`Issued` doesn’t appear anywhere else in the site content.

Comment (quis) [inline src/alert.html:57]:
```suggestion
        Sent by the UK government <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
```

Comment (karlchillmaid) [reply src/alert.html:57]:
```suggestion
        Sent by the UK government <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
```

**Review by benthorner — APPROVED** [review]

**Pull request merged**